packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/DevMenuReactRootViewContainer.kt

```diff
@@ -1,20 +1,57 @@
 package expo.modules.devmenu
 
 import android.content.Context
+import android.graphics.Rect
+import android.os.Build
 import android.view.MotionEvent
 import android.view.View
 import android.view.ViewGroup
 import android.widget.FrameLayout
+import androidx.annotation.RequiresApi
+import androidx.core.view.doOnLayout
 import com.facebook.react.ReactRootView
 import expo.modules.devmenu.fab.MovableFloatingActionButton
 
 /**
  * We want to hide the FAB behind the feature flag for now.
  */
-private const val enableFAB = false
+private const val enableFAB = true
 
 class DevMenuReactRootViewContainer(context: Context) : FrameLayout(context) {
-  private val fab by lazy { MovableFloatingActionButton(context) }
+  @RequiresApi(Build.VERSION_CODES.Q)
+  private val updateSystemGestureExclusionRects: () -> Unit = {
+    val marginLayoutParams = fab.layoutParams as MarginLayoutParams
+
+    // Bounding box for the FAB, with margins included
+    val rect = Rect(
+      fab.x.toInt() - marginLayoutParams.leftMargin,
+      fab.y.toInt() - marginLayoutParams.bottomMargin,
+      fab.x.toInt() + fab.width + marginLayoutParams.rightMargin,
+      fab.y.toInt() + fab.height + marginLayoutParams.topMargin
+    )
+
+    // For some reason, updating the system gesture exclusion rects has to be called on that view
+    // instead of calling it on the fab view itself. Probably, because we want to extend the react by view margins.
+    setSystemGestureExclusionRects(listOf(rect))
+  }
+
+  private val fab by lazy {
+    MovableFloatingActionButton(context) {
+      if (Build.VERSION.SDK_INT < Build.VERSION_CODES.Q) {
+        return@MovableFloatingActionButton
+      }
+
+      // `setSystemGestureExclusionRects` should be call after the view is laid out
+      doOnLayout {
+        updateSystemGestureExclusionRects()
+      }
+    }
+  }
+
+  override fun onLayout(changed: Boolean, left: Int, top: Int, right: Int, bottom: Int) {
+    super.onLayout(changed, left, top, right, bottom)
+    updateSystemGestureExclusionRects()
```

Comment:
```suggestion
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
      updateSystemGestureExclusionRects()
    }
```

not sure if we need to add the check here

Reply:
It is not necessary to check it. It has already been done in the `updateSystemGestureExclusionRects` function.